Bearded.UI/Controls/Control.cs

```diff
@@ -33,15 +45,29 @@ public bool IsVisible
                 else
                 {
                     MadeInvisible();
-                    if (IsFocused) Unfocus();
+                    if (IsFocused)
+                        Unfocus();
                 }
             }
         }
 
         public bool IsClickThrough { get; protected set; }
 
         public bool IsFocused { get; private set; }
-        public bool CanBeFocused { get; protected set; }
+
+        private bool canBeFocused = false;
+        public bool CanBeFocused
+        {
+            get => canBeFocused;
+            protected set
+            {
+                if (!value && IsFocused)
```

Comment:
Is this a behavioural change? If so, please address in a separate PR and explain reasoning why the old method was broken.

Reply:
Will revert and file an issue.

Reply:
The issue: https://github.com/beardgame/ui/issues/21